app/assets/stylesheets/_subnav.scss

```diff
@@ -100,12 +125,29 @@
       margin-right: 0.2em;
     }
   }
+}
 
-  .step--doing {
-    background-color: $color-pale-toffee;
-    color: $color-dark-toffee;
+// When the first stage is in progress (ie: toffee, not green)
+// the left half of the subnav needs to be toffee too.
+.study-subnav--concept {
+  background-color: $color-pale-toffee;
+  @include gradient-horizontal(
+    $start-color: $color-pale-toffee,
+    $end-color: $color-off-white,
+    $start-percent: 50%,
+    $end-percent: 50%
+  );
+}
 
-    @include img-retina("stage-doing.png", "stage-doing@2x.png", 16px, 100px);
-    background-image: url("stage-doing.svg"), none;
-  }
+// When the final stage is in process (ie: toffee, not off-white)
+// the right half of the subnav needs to be toffee too.
+.study-subnav--completion,
```

Comment:
Selector `study-subnav--withdrawn_postponed` should be written in hyphenated BEM (Block Element Modifier) format